api/v1alpha1/nnfcontainerprofile_types.go

```diff
@@ -26,6 +26,14 @@ import (
 
 // NnfContainerProfileSpec defines the desired state of NnfContainerProfile
 type NnfContainerProfileData struct {
+	// Default is true if this instance is the default resource to use
+	// +kubebuilder:default:=false
+	Default bool `json:"default,omitempty"`
```

Comment:
does default make sense for containers?

Reply:
I thought that a default profile that contains only (or mostly) optional storages might be useful. But perhaps not.

Reply:
I'll remove this. We got this for free with @roehrich-hpe's storage profile implementation, so I figured why not.